src/tests/test_pkcs11_high_level.cpp

```diff
@@ -489,9 +489,23 @@ Test::Result test_attribute_container()
    attributes.add_binary(AttributeType::Value, bin);
 
    attributes.add_bool(AttributeType::Sensitive, true);
-   attributes.add_numeric(AttributeType::ObjectId, 12);
-
-   result.test_eq("Five elements in attribute container", attributes.count(), 5);
+   attributes.add_numeric(AttributeType::ObjectId, 10);
+   attributes.add_numeric(AttributeType::Id, 20);
+   attributes.add_numeric(AttributeType::PixelX, 30);
+   attributes.add_numeric(AttributeType::Id, 21);
+   attributes.add_numeric(AttributeType::PixelY, 40);
```

Comment:
Maybe add a comment giving a brief description what this is supposed to check. I assume, that `AttributeType::Id` is successfully overwritten, no?